### DISCUSSION TITLE
HTTPS setup

### DISCUSSION THREAD

**Discussion by Zorbadil** [magicbug/Cloudlog#3053] [General]
Is there a walkthrough available on how to move a cloudlog installation that has been installed as a HTTP site to a HTTPS site.

You can probably tell by my wording that I have little knowlege of what I ask, but I would like to use the API function to update my cloudlog from LOG4OM2, and this does not seem to work under HTTP.

Thanks in advance.

**Comment by phl0** [comment]
API works regardless of HTTP or HTTPS.

**Reply by Zorbadil** [reply]
So I've discovered.  Thank you.
